examples/README.md

```diff
@@ -67,24 +67,22 @@ ansible-playbook node.yml -i examples/inventory-local-genesis.yml -e 'target=SER
 
 Set up a chain with three validator nodes that have the following voting power:
 
-| Validator moniker | Voting power | Self-delegating address                         |
```

Comment:
I'm thinking we should leave this example as-is, but rename it to something like "Start a three-node testnet from existing keys and genesis file".
Then below it we could have another example that uses the dynamic persistent peers and call it "Start a three-node testnet from scratch".
The inventory file would have to be split in two as well.

We should still show the feature of deploying existing keys and genesis files.